packages/clay-empty-state/src/index.tsx

```diff
@@ -59,6 +59,34 @@ const ClayEmptyState = ({
 }: IProps) => {
 	const hasImg = imgSrc || imgProps;
 
+	const [error, setError] = useState(false);
+
+	const reducedMotionImage = useMemo(() => {
+		if (error) {
+			console.warn(
+				'The image url defined in `imgSrcReducedMotion` does not exist on the server. You can provide an updated url through the attribute `imgSrcReducedMotion` or set it to `{null}`.'
+			);
+
+			return null;
+		}
+
+		if (imgSrc && imgSrcReducedMotion !== null) {
+			const url = new URL(
+				imgSrc,
+				imgSrc?.match(/http:\/\/|https:\/\//)
+					? undefined
+					: `https://${location.host}`
+			);
+
+			const index = url.pathname.match(/.(gif|png| jpeg|jpg)/)?.index;
```

Comment:
@pat270 just to avoid use cases where the URL has a link to the image but doesn't have the file extension, we can return null instead of still trying to infer the URL.